examples/standard/multicluster/httproute-gamma.yaml

```
@@ -0,0 +1,16 @@
+#$ Used in:
+#$ - geps/gep-1748.md
+kind: HTTPRoute
+apiVersion: gateway.networking.k8s.io/v1beta1
+metadata:
+  name: store
+spec:
+  parentRefs:
+  - group: multicluster.x-k8s.io
+    kind: ServiceImport
+    name: store
+  rules:
+  - backendRefs:
+    - kind: Service
+      name: store
+      port: 8080
```

This appears to be configuring traffic to a `ServiceImport` ClusterSetIP to instead stay cluster-local, which feels a bit odd (and possibly more natural to configure by either not exporting the service in the first place or blocking `store.svc.clusterset.local` with egress policy). Would the inverse of this make more sense?
```suggestion
spec:
  parentRefs:
  - kind: Service
    name: store
  rules:
  - backendRefs:
    - group: multicluster.x-k8s.io
      kind: ServiceImport
      name: store
```
**Alternatively**, splitting off one path from a `ServiceImport` to a _different_ `ServiceImport` would be an alternate viable pattern if we're explicitly trying to illustrate `ServiceImport` as a `parentRef`.
```suggestion
spec:
  parentRefs:
  - group: multicluster.x-k8s.io
    kind: ServiceImport
    name: store
  rules:
  - matches:
    - path:
      value: "/cart"
    backendRefs:
    - group: multicluster.x-k8s.io
      kind: ServiceImport
      name: cart
```